map/map_loader/config/lanelet2_map_loader.param.yaml

```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
-    lanelet2_map_projector_type: MGRS   # Options: MGRS, UTM
+    lanelet2_map_projector_type: MGRS   # Options: MGRS, UTM, local
```

Comment:
Local is auto-detected so it is not used here.

Reply:
We decided to use local in https://github.com/autowarefoundation/autoware/issues/3427.